src/mdoc/Verifier.ts

```diff
@@ -196,10 +196,13 @@ export class Verifier {
 
     try {
       const deviceKeyRaw = COSEKeyToRAW(deviceKeyCoseKey);
+      const { kty, crv } = COSEKeyToJWK(deviceKeyCoseKey);
       const ephemeralMacKey = await calculateEphemeralMacKey(
         options.ephemeralPrivateKey,
         deviceKeyRaw,
         options.sessionTranscriptBytes,
+        kty,
+        crv,
```

Comment:
`deviceKeyRaw` and `kty`/`crv` are the same COSE key, right?
What if we pass `deviceKeyCoseKey` to the `calculateEphemeralMacKey` function and there is where we convert it to raw/jwk ?